templates/skeleton/server.ts

```diff
@@ -38,6 +44,9 @@ export default {
         HydrogenSession.init(request, [env.SESSION_SECRET]),
       ]);
 
+      const nonce = toHexString(crypto.getRandomValues(new Uint8Array(16)));
+      (request as any).nonce = nonce;
```

Comment:
This is super hacky, though I'm not sure of another way to define the nonce in `server.ts`, and then use it within `entry.server.tsx`. Perhaps when async context becomes available in Cloudflare that could be used? There's also discussion from Remix to [expose the `loadContext` to `entry.server.tsx`](https://github.com/remix-run/remix/discussions/4603).